webknossos-datastore/app/com/scalableminds/webknossos/datastore/datavault/S3DataVault.scala

```diff
@@ -73,6 +75,26 @@ class S3DataVault(s3AccessKeyCredential: Option[S3AccessKeyCredential], uri: URI
     }
   }
 
+  private def performGetObjectSummariesRequest(bucketName: String, keyPrefix: String)(
```

Comment:
The other method was called performRequest, and got a request as a parameter. I’d vote to call this one getObjectSummaries, since it constructs its own request, rather than “performing” a passed one. What do you think?